en/docs/chapter_tree/binary_tree.md

```diff
@@ -1,6 +1,6 @@
 # Binary tree
 
-A "binary tree" is a non-linear data structure that represents the ancestral and descendent relationships, embodying the "divide and conquer" logic. Similar to a linked list, the basic unit of a binary tree is a node, each containing a value, a reference to the left child node, and a reference to the right child node.
+A "binary tree" is a non-linear data structure that represents the hierarchical relationship between ancestors and descendants, embodying the logic of "dividing and conquering." Similar to a linked list, the basic unit of a binary tree is a node, each containing a value, a reference to the left child node, and a reference to the right child node.
```

Comment:
1. Maybe using "derivational relationship" instead of "hierarchical relationship", which can be closer to the new Chinese version.

2. embodying the divide-and-conquer logic of "bifurcation".

Reply:
hi QiLOL thanks for the suggestion,
 for the first part, I think sticking with hierarchical might be better as thats whats commonly been used.
I will change the second part now!